main/components/Button/index.tsx

```diff
@@ -121,6 +120,7 @@ export type Props = {
   loading?: boolean;
   loadingElement?: ReactElement;
   text?: string;
+  rounded?: number;
```

Comment:
I feel `rounded` isn't a good naming. How about a `borderRadius`? Good naming = developers don't need to look up the description.

Reply:
I agree with you!!